### PR TITLE
v4.1.0 - Added new rules module for js classes

### PR DESCRIPTION
**Pull request by oliversweb**
### Added
- Extended the project eslint rule file to cover class variables
---
https://eslint.org/docs/rules/lines-between-class-members

### CODE REVIEW

**Review by kevinrodrigues — APPROVED** [review]

File: rules/style.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 module.exports = {
     rules: {
+        // class specfic styles
+        'lines-between-class-members': [
```
Comment (kevinrodrigues) [inline rules/style.js:4]: Need the package version && changelog too.

**Review by kevinrodrigues — CHANGES_REQUESTED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: rules/style.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
 module.exports = {
     rules: {
+        // class specfic styles
```
Comment (DamianMullins) [inline rules/style.js:3]: Minor thing but the rules in these files look to be alphabetical which does make life easier when you're adding a load of the rules and aren't sure where to place them in the module (I'm guessing makes it easier to find a particular rule?). If you want to create a new section for classes maybe we could create a new module?

**Review by kevinrodrigues — APPROVED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Review by ashleynolan — APPROVED** [review]

**Review by ashleynolan — CHANGES_REQUESTED** [review]
Just ran a couple of the build checks locally (as it looks like this hooks up to travis which isn't running) – looks like running `yarn lint` on this throws up a few errors (minor things).

Could you fix these?

**Comment by ashleynolan** [comment]
I've updated the linting rules now for this – going to look to see if I can switch the build on Circle shortly too so this runs automatically.

**Pull request merged**